Add solar radiation components #1076

l-emele opened the pull request:
# Summary of the discussion

Add subclasses of solar radiation and add concepts related to solar tracking.

## Type of change (CHANGELOG.md)

### Added
* `direct solar radiation`
* `diffuse solar radiation`
* `non-scattered radiant flux density`
* `solar tracking`
* `single axis tracking`
* `two axis tracking`
* `solar tracked receiving surface`
* `single axis tracked receiving surface`
* `two axis tracked receiving surface`

### Updated
Add editor note to `solar radiation`

## Workflow checklist

### Automation
No automation, implements only parts of #1076.

### PR-Assignee
- [ ] 🐙 Follow the [Pull Request Workflow](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow)
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CHANGELOG.md)
- [ ] 📙 Add #'s to `term tracker item`

### Reviewer
- [ ] 🐙 Follow the [Reviewer Guide](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow#reviewer-guide-check-changes-introduced-by-a-pull-request)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

carstenhoyerklick reviewed (changes requested):
Thanks a lot, I am missing two small things: in #1076 we said to place global radiation as an alternative term for solar radiation and we wanted to add an axiom 'not scattered radiant flux density' 'is about' some 'direct radiation'. Did I miss it or should we add this?

l-emele commented:
I added the alternative label `global radiation` now. I added the axiom with a small modification: `'not scattered radiant flux density' 'quantity value of' some 'direct radiation'`. The object property `quantity value of` is a subproperty of `is about`.

carstenhoyerklick commented:
Thanks, it is now fine with me.

l-emele commented:
@stap-m : Can you please have a second look on this review?

stap-m commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -6635,6 +6635,135 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1445",
+Class: OEO_00010395
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Solar tracking is a process of in which a radiation receiving surface is following the path of the sun on the sky.",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Solar tracking is a process in which a radiation receiving surface is following the path of the sun on the sky.",
```

stap-m commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -6635,6 +6635,135 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1445",
+Class: OEO_00010392
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Direct solar radiation is radiation that is the non-scattered part of solar radiation from the Sun within the extent of the solar disk only.",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Direct solar radiation is radiation that is the non-scattered part of solar radiation from the sun within the extent of the solar disk only.",
```

stap-m commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -6635,6 +6635,135 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1445",
+Class: OEO_00010397
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Two axis tracking is solar tracking in which a radiation receiving surface is following the sun by rotating around two axes. The plane is always oriented normal.",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Two axis tracking is solar tracking in which a radiation receiving surface is following the sun by rotating around two axes. The surface is always oriented normal.",
```
Is this what is meant?

stap-m commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -6773,8 +6902,14 @@ Class: OEO_00020038
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
         <http://purl.obolibrary.org/obo/IAO_0000115> "Solar radiation is radiation that is emitted by the sun.",
+        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the merological sum of direct solar radiation and diffuse solar radiation",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the merological sum of direct solar radiation and diffuse solar radiation",
```
Is "merological sum" a typo or something that exists?

l-emele replied:
Its a typo, it should be **mereological**.

stap-m reviewed (commented):
Just some minor remarks. Rest look fine to me.

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -6773,8 +6902,14 @@ Class: OEO_00020038
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
         <http://purl.obolibrary.org/obo/IAO_0000115> "Solar radiation is radiation that is emitted by the sun.",
+        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the merological sum of direct solar radiation and diffuse solar radiation",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the mereological sum of direct solar radiation and diffuse solar radiation",
```
